modules/002-deckhouse/hooks/telemetry_update_window.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2022 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hooks
+
+import (
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+	"github.com/flant/addon-operator/sdk"
+
+	"github.com/deckhouse/deckhouse/go_lib/hooks/update"
+	"github.com/deckhouse/deckhouse/go_lib/telemetry"
+)
+
+var _ = sdk.RegisterFunc(&go_hook.HookConfig{
+	OnAfterHelm: &go_hook.OrderedConfig{Order: 1000},
+	// telemetry should not block Deckhouse
+	AllowFailure: true,
+}, updateWindowTelemetry)
+
+func updateWindowTelemetry(input *go_hook.HookInput) error {
```

Comment:
This hook seems like a copy of https://github.com/deckhouse/deckhouse/blob/main/modules/002-deckhouse/hooks/metrics.go
Consider merging these two hooks.

Possible plan:
1. Add prefixes to metrics of the metrics.go hook
2. Check where are they used and change their names accordingly